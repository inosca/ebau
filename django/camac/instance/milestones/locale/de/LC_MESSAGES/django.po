# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: \n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-09-11 17:25+0200\n"
"PO-Revision-Date: 2024-08-29 13:33+0200\n"
"Last-Translator: \n"
"Language-Team: \n"
"Language: de\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: Poedit 3.0.1\n"

#: camac/instance/milestones/serializers.py
msgid "regular process"
msgstr "Ordentliches Verfahren"

#: camac/instance/milestones/serializers.py
msgid "instance submitted"
msgstr "Dossier erfasst"

#: camac/instance/milestones/serializers.py
msgid "Paper submission"
msgstr "Eingang Post"

#: camac/instance/milestones/serializers.py
msgid "Additional demand"
msgstr "Nachforderung Gemeindebaubehörde"

#: camac/instance/milestones/serializers.py
msgid "Check completed"
msgstr "Vollständigkeitsprüfung abgeschlossen"

#: camac/instance/milestones/serializers.py
msgid "Instance complete"
msgstr "Dossier vollständig"

#: camac/instance/milestones/serializers.py
msgid "Review building commission"
msgstr "Prüfung durch Gemeinde"

#: camac/instance/milestones/serializers.py
msgid "Puclication date"
msgstr "Publikationsdatum"

#: camac/instance/milestones/serializers.py
msgid "Receipt of objections"
msgstr "Eingang von Einsprachen"

#: camac/instance/milestones/serializers.py
msgid "Forwarding to KOOR"
msgstr "Weiterleitung an KOOR"

#: camac/instance/milestones/serializers.py
msgid "Start circulation"
msgstr "Start Zirkulation"

#: camac/instance/milestones/serializers.py
msgid "Distribution completed"
msgstr "Zirkulation beendet"

#: camac/instance/milestones/serializers.py
msgid "Statement - preliminary decision"
msgstr "Stellungnahme - Vorentscheid"

#: camac/instance/milestones/serializers.py
msgid "Building decision"
msgstr "Bau- und Einspracheentscheid"

#: camac/instance/milestones/serializers.py
msgid "notice to geometer"
msgstr "Meldung Baubewilligung an Geometer"

#: camac/instance/milestones/serializers.py
msgid "Receipt confirmation of decision documents"
msgstr "Empfangsbestätigung Entscheiddokumente"

#: camac/instance/milestones/serializers.py
msgid "Receipt confirmation of preliminary decision"
msgstr "Empfangsbestätigung Vorentscheid"

#: camac/instance/milestones/serializers.py
msgid "notice to gebaeudeschaetzung"
msgstr "Meldung Baubewilligung an Gebäudeschätzung"

#: camac/instance/milestones/serializers.py
msgid "Building permit valid until"
msgstr "Baubewilligung gültig bis"
