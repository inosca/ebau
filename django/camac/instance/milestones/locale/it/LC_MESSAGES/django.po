# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-09-11 17:25+0200\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: camac/instance/milestones/serializers.py
msgid "regular process"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "instance submitted"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Paper submission"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Additional demand"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Check completed"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Instance complete"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Review building commission"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Publication date"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Receipt of objections"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Forwarding to KOOR"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Start circulation"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Distribution completed"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Statement - preliminary decision"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Building decision"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "notice to geometer"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Receipt confirmation of decision documents"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Receipt confirmation of preliminary decision"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "notice to gebaeudeschaetzung"
msgstr ""

#: camac/instance/milestones/serializers.py
msgid "Building permit valid until"
msgstr ""
