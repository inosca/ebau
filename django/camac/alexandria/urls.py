from alexandria.core import views as alexandria_views
from django.urls import path
from rest_framework.routers import SimpleRouter

from camac.alexandria import views

r = SimpleRouter(trailing_slash=False)

# override the default viewset with our patched versions
# this needs to be updated when new urls are added in alexandria
r.register(r"categories", views.PatchedCategoryViewSet)
r.register(r"documents", views.PatchedDocumentViewSet)
r.register(r"files", views.PatchedFileViewSet)
r.register(r"tags", views.PatchedTagViewSet)
r.register(r"tagsynonymgroups", views.PatchedTagSynonymGroupViewSet)
r.register(r"marks", views.PatchedMarkViewSet)

urlpatterns = r.urls

urlpatterns.append(path("hook", alexandria_views.hook_view, name="hook"))
