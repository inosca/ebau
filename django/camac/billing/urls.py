from django.urls import re_path
from rest_framework.routers import SimpleRouter

from camac.billing import views
from camac.billing.export.views import BillingV2EntryExportView

urlpatterns = [
    re_path(
        r"billing-v2-entries/export$",
        BillingV2EntryExportView.as_view(),
        name="billing-export",
    ),
]

r = SimpleRouter(trailing_slash=False)

r.register(r"billing-v2-entries", views.BillingV2EntryViewset, "billing-v2-entry")

urlpatterns.extend(r.urls)
