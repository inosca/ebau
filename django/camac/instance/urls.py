from django.conf import settings
from django.urls import re_path
from rest_framework.routers import SimpleRouter

from . import views


class UnswaggeredInstanceView(views.InstanceView):
    swagger_schema = None


r = SimpleRouter(trailing_slash=False)
urlpatterns = []

r.register(r"instance-states", views.InstanceStateView, "instance-state")
r.register(r"forms", views.FormView, "form")
r.register(r"instances", views.InstanceView, "instance")
r.register(r"form-fields", views.FormFieldView, "form-field")
r.register(
    r"instance-responsibilities",
    views.InstanceResponsibilityView,
    "instance-responsibility",
)
r.register(r"journal-entries", views.JournalEntryView, "journal-entry")
r.register(r"history-entries", views.HistoryEntryView, "history-entry")
r.register(r"issues", views.IssueView)
r.register(r"issue-templates", views.IssueTemplateView, "issue-template")
r.register(r"issue-template-sets", views.IssueTemplateSetView, "issue-template-set")

urlpatterns.append(
    re_path(
        r"form-config",
        views.FormConfigDownloadView.as_view(),
        name="form-config-download",
    )
)


def enable_public_caluma_instances():
    urlpatterns.append(
        re_path(
            r"public-caluma-instances",
            views.PublicCalumaInstanceView.as_view(),
            name="public-caluma-instance",
        )
    )


if settings.APPLICATION.get("ENABLE_PUBLIC_ENDPOINTS"):  # pragma: no cover
    enable_public_caluma_instances()

urlpatterns.extend(r.urls)
