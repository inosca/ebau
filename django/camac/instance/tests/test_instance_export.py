import pathlib

import pyexcel
import pytest
from caluma.caluma_form import api as form_api, models as caluma_form_models
from django.urls import reverse
from rest_framework import status


@pytest.mark.parametrize("role__name", ["Municipality"])
@pytest.mark.parametrize("service__name", ["Leitbeh√∂rde Burgdorf"])
def test_caluma_export_be(
    db,
    admin_client,
    be_instance,
    instance_service_factory,
    service,
    application_settings,
    settings,
    django_assert_num_queries,
):
    settings.APPLICATION_NAME = "kt_bern"
    application_settings["MUNICIPALITY_DATA_SHEET"] = settings.ROOT_DIR(
        "kt_bern",
        pathlib.Path(settings.APPLICATIONS["kt_bern"]["MUNICIPALITY_DATA_SHEET"]).name,
    )

    instance_service_factory(
        instance=be_instance, service=admin_client.user.groups.first().service
    )

    be_instance.case.document.answers.create(
        value=str(service.pk), question_id="gemeinde"
    )

    row_doc = form_api.save_document(
        caluma_form_models.Form.objects.get(pk="personalien-tabelle")
    )
    form_api.save_answer(
        caluma_form_models.Question.objects.get(pk="vorname-gesuchstellerin"),
        row_doc,
        value="Max",
    )
    form_api.save_answer(
        caluma_form_models.Question.objects.get(pk="name-gesuchstellerin"),
        row_doc,
        value="Muster",
    )

    form_api.save_answer(
        caluma_form_models.Question.objects.get(pk="personalien-gesuchstellerin"),
        be_instance.case.document,
        value=[str(row_doc.pk)],
    )

    url = reverse("instance-export")

    with django_assert_num_queries(2):
        response = admin_client.get(url, {"instance_id": be_instance.pk})

    assert response.status_code == status.HTTP_200_OK
    book = pyexcel.get_book(file_content=response.content, file_type="xlsx")
    assert be_instance.pk in book.get_dict()["pyexcel sheet"][1]


@pytest.mark.parametrize(
    "query",
    [
        {},
        {"foo": "bar"},
        {"instance_id": ""},
        {"instance_id": ",".join(str(i) for i in range(10000, 11001))},
    ],
)
def test_caluma_export_bad_request(admin_client, query):
    url = reverse("instance-export")
    resp = admin_client.get(url, query)

    assert resp.status_code == status.HTTP_400_BAD_REQUEST
