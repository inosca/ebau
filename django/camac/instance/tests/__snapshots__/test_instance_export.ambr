# serializer version: 1
# name: test_caluma_export_sz[False-Gemeinde Schwyz-Municipality-200-1-3]
  list([
    '123-45-77',
    'Test form',
    'Test location',
    'Smoothie-licious Inc., Red Apple',
    'Test intent',
    'Test location',
    'Test instance state',
    '',
    '02.03.2023',
    '',
    '',
    '',
    '01.04.2023',
    '03.04.2023',
  ])
# ---
# name: test_caluma_export_sz[False-Gemeinde Schwyz-Service-200-1-3]
  list([
    '123-45-77',
    'Test form',
    'Test location',
    'Smoothie-licious Inc., Red Apple',
    'Test intent',
    'Test location',
    'Test instance state',
    '',
    '02.03.2023',
    '',
    '',
    '',
    '01.04.2023',
    '03.04.2023',
  ])
# ---
# name: test_caluma_export_sz[True-Gemeinde Schwyz-Municipality-200-1-3]
  list([
    '123-45-77',
    'Test form',
    'Test location',
    'Smoothie-not-so-licious Inc., Red Apple',
    'Test intent override',
    'Test address, Test special name, Test location',
    'Test instance state',
    '',
    '02.03.2023',
    '',
    '',
    '',
    '01.04.2023',
    '03.04.2023',
  ])
# ---
# name: test_caluma_export_sz[True-Gemeinde Schwyz-Service-200-1-3]
  list([
    '123-45-77',
    'Test form',
    'Test location',
    'Smoothie-not-so-licious Inc., Red Apple',
    'Test intent override',
    'Test address, Test special name, Test location',
    'Test instance state',
    '',
    '02.03.2023',
    '',
    '',
    '',
    '01.04.2023',
    '03.04.2023',
  ])
# ---
